code/game/gamemodes/umbrage/umbrage_minor_abilities.dm

```diff
@@ -0,0 +1,284 @@
+	<span class='danger'>You begin harnessing every ounce of your power...</span>")
+	playsound(usr, 'sound/magic/demented_outburst_charge.ogg', 100, 0)
+	addtimer(CALLBACK(src, .proc/outburst, usr), 50)
+	return 1
+
+/datum/action/innate/umbrage/demented_outburst/proc/outburst(mob/living/user)
+	if(!user || user.stat)
+		return
+	user.visible_message("<span class='boldwarning'>[user] lets out a deafening scream!</span>", "<span class='velvet bold' italics'>WSWU!</span><br>\
+	<span class='danger'>You let out a deafening outburst!</span>")
+	playsound(user, 'sound/magic/demented_outburst_scream.ogg', 150, 0)
+	var/list/thrown_atoms = list()
+	for(var/turf/T in view(5, user))
+		for(var/atom/movable/AM in T)
+			thrown_atoms += AM
+	for(var/atom/movable/AM in thrown_atoms)
+		if(AM == user || AM.anchored)
+			continue
+		var/distance = get_dist(user, AM)
+		var/turf/target = get_edge_target_turf(user, get_dir(user, get_step_away(AM, user)))
+		AM.throw_at(target, ((Clamp((5 - (Clamp(distance - 2, 0, distance))), 3, 5))), 1, user)
+		if(iscarbon(AM))
+			var/mob/living/carbon/L = AM
+			if(distance <= 1) //you done fucked up now
+				L.visible_message("<span class='warning'>The blast sends [L] flying!</span>", "<span class='userdanger'>The force sends you flying!</span>")
+				L.Weaken(5)
+				L.adjustBruteLoss(10)
+				L.soundbang_act(1, 5, 15, 5)
+			else if(distance <= 3)
+				L.visible_message("<span class='warning'>The blast knocks [L] off their feet!</span>", "<span class='userdanger'>The force bowls you over!</span>")
+				L.Weaken(3)
+				L.soundbang_act(1, 3, 5, 0)
+	return 1
+
+
+//Simulacrum: Creates an illusionary copy of the umbrage that moves continually in the direction that they're facing.
+// - The illusion lasts for ten seconds and cannot be attacked.
+// - If someone examines the illusion, they will be able to tell that it's false.
+/datum/action/innate/umbrage/simulacrum
+	name = "Simulacrum"
+	id = "simulacrum"
+	desc = "Creates an illusion that closely resembles you. The illusion will run forward for ten seconds."
+	button_icon_state = "umbrage_simulacrum"
+	check_flags = AB_CHECK_STUNNED|AB_CHECK_CONSCIOUS
+	psi_cost = 30
+	lucidity_cost = 1
+	blacklisted = 0
+
+/datum/action/innate/umbrage/simulacrum/Activate()
+	usr.visible_message("<span class='warning'>[usr] suddenly splits into two!</span>", "<span class='velvet bold'>zayaera</span><br>\
+	<span class='notice'>You create an illusion of yourself.</span>")
+	playsound(usr, 'sound/magic/devour_will_form.ogg', 50, 1)
+	var/obj/effect/simulacrum/simulacrum = new(get_turf(usr))
+	simulacrum.mimic(usr)
+	return 1
+
+
+//Tagalong: Melds with a conscious mob's shadow, allowing the umbrage to freely accompany them anywhere they go.
+// - Only usable in targets not in full darkness.
+// - If the target enters a fully dark area, the umbrage will be stunned and ejected.
+/datum/action/innate/umbrage/tagalong
```

Comment:
I might've, actually. It seemed fitting to me so I just used it, but I didn't consciously take it.
Being is still on the backburner, don't give up on me!